### PR TITLE
Enhance Chat Mentions

### PR DESCRIPTION
**Pull request by odkhang**
> Fixes [#138  Enhance Chat Mentions](https://github.com/fossasia/eventyay-video/issues/138)
> 
> #### Short description of what this resolves:
> Implement @username mentions in the chat 
> 
> #### Changes proposed in this pull request:
> * Implement @username mentions in the chat
> 
> #### Checklist
> * [x]  I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
> * [x]  My branch is up-to-date with the Upstream `development` branch.
> * [ ]  The acceptance, integration, unit tests and linter pass locally with my changes
> * [ ]  I have added tests that prove my fix is effective or that my feature works
> * [ ]  I have added necessary documentation (if appropriate)

### CODE REVIEW

File: server/tests/live/test_auth.py [hunk 1]
```diff
@@ -1062,3 +1073,4 @@ async def test_anonymous_invite(client, world, stream_room, bbb_room):
             "room:poll.vote",
             "room:poll.read",
         }
+
```
Comment (hongquan) [inline server/tests/live/test_auth.py:1076]: Please configure your editor to automatically clean trailing whitespaces (at the end of line).

File: server/tests/live/test_chat.py [hunk 2]
```diff
@@ -1324,3 +1324,4 @@ async def test_force_join_after_login(world, chat_room):
             r = await c2.receive_json_from()
         assert r[0] == "chat.channels"
         assert channel_id in [c["id"] for c in r[1]["channels"]]
+
```
Comment (hongquan) [inline server/tests/live/test_chat.py:1327]: Clean trailing whitespaces, too.

File: server/tests/live/test_chat_direct.py [hunk 3]
```diff
@@ -787,3 +791,207 @@ async def test_send_if_silenced(world):
         response = await c1.receive_json_from()
         assert "error" == response[0]
         assert "chat.denied" == response[2]["code"]
+
```
Comment (hongquan) [inline server/tests/live/test_chat_direct.py:794]: 2 blank lines between functions, classes (PEP8 coding style).

**Review by hongquan — CHANGES_REQUESTED** [review]

File: server/venueless/core/migrations/0065_alter_auditlog_data_alter_chatevent_content_and_more.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0064_auto_20230805_1737"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="auditlog",
```
Comment (hongquan) [inline server/venueless/core/migrations/0065_alter_auditlog_data_alter_chatevent_content_and_more.py:17]: Why are there migrations relatd to `AuditLog` model? I don't see changes of `AuditLog` model in this PR.

**Comment by odkhang** [comment]
Hi @hongquan, 
I removed unintended changes in the AuditLog model by deleting the incorrect migration file and regenerating the migrations. Now, the new migration file only includes the intended updates.
Please review the updated migrations.

**Review by hongquan — APPROVED** [review]

**Pull request merged**